lms/static/styles/components/_StudentSelector.scss

```diff
@@ -1,29 +1,41 @@
 .StudentSelector {
   display: flex;
   button {
-    background-color: transparent;
-    min-width: 80px;
+    svg {
+      color: $grey-5;
+    }
+    min-width: 60px;
+    height: 40px;
     border: none;
-    background-color: $grey-6;
-    transition: background-color 0.2s, opacity 0.2s;
-    outline: none;
+    background-color: $grey-1;
+    border: 1px solid $grey-3;
+    transition: 0.2s;
+    z-index: 1; /* prevents focus outline from rendering under select */
+
+    &:focus {
+      border-color: $grey-5;
+    }
     &:hover {
       cursor: pointer;
-      background-color: $grey-5;
+      background-color: $grey-2;
     }
     &:disabled {
-      &:hover {
-        background-color: $grey-6;
-      }
+      background-color: $grey-1;
       cursor: default;
       opacity: 0.5;
     }
   }
 }
 
 .StudentsSelector__student {
-  padding: 10px 20px;
-  display: flex;
-  justify-content: space-around;
-  flex-flow: column;
+  select {
+    appearance: none;
```

Comment:
Doesn't look like `appearance` is supported in IE11. Can you verify that this does what you'd expect in IE 11?